ShopifySharp/Entities/ServiceFilter.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShopifySharp
+{
+    public class ServiceFilter
```

Comment:
I lean toward removing this class since it's not documented by Shopify. Are you using this in your own implementation?